AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/MAX/Banners/ALMAXAutoLayoutBannerAdViewController.m

```diff
@@ -24,7 +24,6 @@ - (void)viewDidLoad
     self.adView = [[MAAdView alloc] initWithAdUnitIdentifier: @"YOUR_AD_UNIT_ID"];
     
     self.adView.delegate = self;
-    self.adView.revenueDelegate = self;
```

Comment:
Actually, this is not a deprecated property so let's revert all `MAAdRevenueDelegate` changes. The deprecation warning should only be for implementation of the callback via `MAAdDelegate`. Both delegates contain the `- (void)didPayRevenueForAd:(MAAd *)ad` method.